Client.php

```diff
@@ -78,21 +77,21 @@ class Client extends Connection {
     /**
      * Create a Websocket client.
      *
-     * @access  public
      * @param   \Hoa\Socket\Client  $client      Client.
      * @param   string              $endPoint    End-point.
      * @param   \Hoa\Http\Response  $request     Response parser.
      * @return  void
      * @throw   \Hoa\Socket\Exception
      */
-    public function __construct ( Socket\Client $client, $endPoint = '/',
-                                  Http\Response $response = null ) {
-
+    public function __construct(Socket\Client $client, $endPoint = '/',
+                                Http\Response $response = null)
+    {
```

Comment:
If I am correct, it will be like 

```php
public function __construct(
    Socket\Client $client, $endPoint = '/',
    Http\Response $response = null
) {
```